[CI] Failure in SuggestStatsIT testSimpleStats

Link: https://elasticsearch-ci.elastic.co/job/elastic+elasticsearch+master+multijob-unix-compatibility/os=centos/2018/console

```
00:24:36 FAILURE 4.45s J1 | SuggestStatsIT.testSimpleStats <<< FAILURES!
00:24:36    > Throwable #1: java.lang.AssertionError: 
00:24:36    > Expected: a value greater than <0L>
00:24:36    >      but: <0L> was equal to <0L>
00:24:36    > 	at __randomizedtesting.SeedInfo.seed([FDC21049ED079812:B806696314BB0DE2]:0)
00:24:36    > 	at org.hamcrest.MatcherAssert.assertThat(MatcherAssert.java:20)
00:24:36    > 	at org.elasticsearch.index.suggest.stats.SuggestStatsIT.testSimpleStats(SuggestStatsIT.java:127)
00:24:36    > 	at java.lang.Thread.run(Thread.java:748)
```

Doesn't reproduce locally for me so far with:
```
./gradlew :server:integTest -Dtests.seed=FDC21049ED079812 -Dtests.class=org.elasticsearch.index.suggest.stats.SuggestStatsIT -Dtests.method="testSimpleStats" -Dtests.security.manager=true -Dtests.locale=ar-BH -Dtests.timezone=Australia/Eucla
```

--- a/server/src/test/java/org/elasticsearch/index/suggest/stats/SuggestStatsIT.java
+++ b/server/src/test/java/org/elasticsearch/index/suggest/stats/SuggestStatsIT.java
@@ -112,7 +112,7 @@ public class SuggestStatsIT extends ESIntegTestCase {
 
         logger.info("iter {}, iter1 {}, iter2 {}, {}", suggestAllIdx, suggestIdx1, suggestIdx2, endTime - startTime);
         // check suggest time
-        assertThat(suggest.getSuggestTimeInMillis(), greaterThan(0L));
+        assertThat(suggest.getSuggestTimeInMillis(), greaterThanOrEqualTo(0L));
         // the upperbound is num shards * total time since we do searches in parallel
         assertThat(suggest.getSuggestTimeInMillis(), lessThanOrEqualTo(totalShards * (endTime - startTime)));
 
@@ -124,7 +124,7 @@ public class SuggestStatsIT extends ESIntegTestCase {
             logger.info("evaluating {}", stat.getNode());
             if (nodeIdsWithIndex.contains(stat.getNode().getId())) {
                 assertThat(suggestStats.getSuggestCount(), greaterThan(0L));
-                assertThat(suggestStats.getSuggestTimeInMillis(), greaterThan(0L));
+                assertThat(suggestStats.getSuggestTimeInMillis(), greaterThanOrEqualTo(0L));
                 num++;
             } else {
                 assertThat(suggestStats.getSuggestCount(), equalTo(0L));

